.github/workflows/test.yml

```diff
@@ -0,0 +1,66 @@
+name: tests
+
+on: [push, pull_request]
+
+jobs:
+  code-analysis:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up JDK 8
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'adopt'
+          java-version: '8'
+      - name: Cache Maven packages
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-m2-ca-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-m2-ca-
+      - name: Code Analysis
+        run: mvn --batch-mode --activate-profiles github install pmd:check pmd:cpd-check license:check javadoc:jar -DskipTests=true
+  unit-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up JDK 8
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'adopt'
+          java-version: '8'
+      - name: Cache Maven packages
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-m2-ut-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-m2-ut-
+      - name: Unit Tests
+        run: mvn --batch-mode --activate-profiles github test jacoco:report
+      - name: Upload test coverage report
+        uses: codecov/codecov-action@v2
+        with:
+          files: ./ecaudit/target/site/jacoco/jacoco.xml,./common/target/site/jacoco/jacoco.xml,./eclog/target/site/jacoco/jacoco.xml
```

Comment:
You could do something like this:
./**/target/site/jacoco/jacoco.xml

Reply:
Actually, I tried this with different variations (peeking at the example in ecChronos repo). But the action wouldn't pick up the files for some reason. I did notice that ecChronos repo is using v1 of the codecov-action. Is the wildcard support perhaps missing in v2?